app/assets/javascripts/snowcrash/pages/nodes/notes/table.js.coffee

```diff
@@ -0,0 +1,4 @@
+document.addEventListener "turbolinks:load", ->
+  if $('body.nodes.show').length
```

Comment:
@dormi I don't think this will work once we have multiple tabs in the nodes view because there will be multiple index tables rendered in the page. Do you think we should address this now or should we cross the bridge once we get there?

Reply:
let's work on this when we have the problem, probably when dealing with the evidence or attachments tab